### DISCUSSION TITLE
How to move View to edge bottom

### DISCUSSION THREAD

**Discussion by mirik999** [zoontek/react-native-edge-to-edge#32] [Q&A] [answered]
I have installed this package on expo dev build and created a new build, removed expo status bar and navigation bar from codebase and added single <SystemBars ... /> from this library. Also I am not using a safe area. But for the position absolute and bottom 0 it sticks on top of navigation bars like as always. Without hiding the navigation bar how to achieve this behavior ?

**Comment by zoontek** [comment]
@mirik999 Did you [added the plugin](https://github.com/zoontek/react-native-edge-to-edge?tab=readme-ov-file#expo)? Could you provide a reproduction?

**Reply by mirik999** [reply]
Yes, plugin plugged in. 

   [
      ... a bunch of plugins,
      'react-native-edge-to-edge'
    ]
    
    Okay, i will create a new project and will provide a reproduction.

**Reply by mirik999** [reply]
Oh , i just noticed this 

on pixel simulator it works 
![simulator](https://github.com/user-attachments/assets/20f35694-0a01-4fd7-b5d4-a23f4d7fc295)

but on old physical device it doesnt 
![physical-device](https://github.com/user-attachments/assets/93c195e4-1a35-4ac0-8b22-65d51460221b)

**Reply by zoontek** [reply]
I just created a fresh Expo project, installed latest version and build it, it worked correctly:

<img width="1787" alt="Screenshot 2024-10-14 at 14 31 18" src="https://github.com/user-attachments/assets/35f1fad8-e980-476f-8653-b34e2c8c7c12">

**Reply by zoontek** [reply, marked as answer]
@mirik999 On your device, it has fallback to "light scrim", because you are using button navigation in light mode (`#fff`, 90% opacity).

It's visible, but you should be able to draw under. See, if I switch to button navigation on the simulator:

<img width="1790" alt="Screenshot 2024-10-14 at 14 44 01" src="https://github.com/user-attachments/assets/4f538d9c-4a1e-43e7-b60e-33d355f3d7b4">

Fully transparent is only available with gesture navigation, as automatic navigation bar style (based on contrast) also only work with gesture navigation, and it's not possible to programmaticaly update navigation bar style when targetting SDK 35.

[See the AndroidX core edge-to-edge implementation, that does the same thing.](https://github.com/androidx/androidx/blob/26388331a624607466d4accea62518ceb6dfccfb/activity/activity/src/main/java/androidx/activity/EdgeToEdge.kt#L71) ([the light scrim](https://github.com/androidx/androidx/blob/androidx-main/activity/activity/src/main/java/androidx/activity/EdgeToEdge.kt#L37))
